README.md

```diff
@@ -90,20 +92,6 @@ responses:
             type: string
             xml:
               name: photoUrl
-        tags:
-          type: array
-          xml:
-            wrapped: true
-          items:
-            type: object
-            properties:
-              id:
-                type: integer
-                format: int64
-              name:
-                type: string
-            xml:
-              name: Tag
```

Comment:
I removed the `tags` section of the readme toy example. It's still really long, but was **really** long